libs/fluid-elements/button/tsconfig.spec.json

```
@@ -0,0 +1,15 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../../dist/out-tsc",
+    "module": "commonjs",
+    "types": ["jest", "node"]
+  },
+  "include": [
+    "**/*.spec.ts",
+    "**/*.spec.tsx",
+    "**/*.spec.js",
+    "**/*.spec.jsx",
+    "**/*.d.ts"
```

We don't have jsx so please remove

```suggestion
  "include": [
    "**/*.spec.ts",
    "**/*.d.ts"
```